### DISCUSSION TITLE
What 'minus' mean?

### DISCUSSION THREAD

**Discussion by KES777** [dbeaver/dbeaver#13013] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/799021/123438789-bc609d80-d5d9-11eb-97f3-b75053a3926c.png)

When I select item DBeaver put minus sign. What does this mean: item is removed from processing?

Please use 'ok sign' like `v`

**Comment by ShadelessFox** [comment, marked as answer]
Hello @KES777,

This `minus` sign is an intermediate state, denoting that not all of the child elements of this node are selected, and the icon itself comes from your OS theme. This is a default behavior for tree-like widgets.
